source/common/http/utility.cc

```diff
@@ -101,18 +101,18 @@ std::string Utility::parseCookieValue(const HeaderMap& headers, const std::strin
               // checking other cookies in this header.
               continue;
             }
-            std::string k = s.substr(first_non_space, equals_index - first_non_space);
+            absl::string_view k = s.substr(first_non_space, equals_index - first_non_space);
```

Comment:
const here and below

Reply:
```absl::string_view v``` might need to be modified, we can't make it a const:
```
if (v.size() >= 2 && v.back() == '"' && v[0] == '"') {
      v = v.substr(1, v.size() - 2);
}
```

Reply:
Cool no problem, sorry for the invalid comment.